src/config/src/file-store.ts

```diff
@@ -0,0 +1,83 @@
+import { Store } from './store'
+import { existsSync, writeFile, readFileSync } from 'node:fs'
+import { randomUUID } from 'node:crypto'
+import { MINIMUM_UPDATE_ON_RESTART_COOLDOWN } from './constants'
+
+const file = './codeclimbers.identity'
+
+export class FileStore implements Store {
```

Comment:
Nice implementation! Looks like this should work well. curious why you chose to write this to file instead of the db? It does feel weird to create an entire table to store and retrieve one record

Reply:
The main reason was preventing the server and config locking each other out of the database by either both accessing the file or by them cross-depending on each other to start up.